admins/pageflow/entry.rb

```diff
@@ -48,7 +51,7 @@ module Pageflow
             f.input :theming, :include_blank => false
           end
         end
-        if authorized?(:manage, Folder)
+        if authorized?(:configure_folder_for, resource || resource_class)
```

Comment:
If `resource` is `nil`, the check `authorized?(:configure_folder_for,, Entry)` will be performed, which will always be true. Can we make it more explicit, that the folder fields should always be visible if we are creating a new entry (which is the only case where `resource` can be `nil` here).